### DISCUSSION TITLE
grid-layout

### DISCUSSION THREAD

**Discussion by corbrink** [Madelena/Metrology-for-Hass#64] [General]
Hi Madelena,
This theme and work is great, thank you for sharing. I notice that his can only be implemented in yaml mode and not lovelace mode, correct?
One more question please - how do you implement the grid-layout and other css files? I cannot get the astrometrics view to work (don't know about the other views yet) because of the following message: "Custom element doesn't exist: grid-layout.

Thank you & regards

**Comment by Madelena** [comment]
This theme can be used in YAML or Lovelace mode, especially in 2.0 where cards that come with HA are also themed.

`grid-layout` comes from [`layout-card` by @thomasloven](https://github.com/thomasloven/lovelace-layout-card/). It allows cards to be laid out using CSS Grid logic. You can use this can in YAML mode once you have this frontend extension installed.
